src/packages/@workspaces/features/testflow-explorer/icons/Run.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
+  export let height = 7;
+  export let width = 7;
+  export let color = "white";
+</script>
+
+<svg
+  {width}
+  {height}
+  viewBox="0 0 12 14"
+  fill="none"
+  xmlns="http://www.w3.org/2000/svg"
+>
+  <path
+    d="M0.9375 12.1742V1.82422C0.9375 1.54089 1.0375 1.30339 1.2375 1.11172C1.4375 0.920052 1.67083 0.824219 1.9375 0.824219C2.02083 0.824219 2.10833 0.836719 2.2 0.861719C2.29167 0.886719 2.37917 0.924219 2.4625 0.974219L10.6125 6.14922C10.7625 6.24922 10.875 6.37422 10.95 6.52422C11.025 6.67422 11.0625 6.83255 11.0625 6.99922C11.0625 7.16589 11.025 7.32422 10.95 7.47422C10.875 7.62422 10.7625 7.74922 10.6125 7.84922L2.4625 13.0242C2.37917 13.0742 2.29167 13.1117 2.2 13.1367C2.10833 13.1617 2.02083 13.1742 1.9375 13.1742C1.67083 13.1742 1.4375 13.0784 1.2375 12.8867C1.0375 12.6951 0.9375 12.4576 0.9375 12.1742Z"
+    fill={color}
+  />
+</svg>
```

Comment:
push this to library, this may be used in overall app, and use this script
`<script>
  export let height = "24px";
  export let width = "24px";
  export let color = "grey";
</script>`